### PR TITLE
{bio,lib}[intel/2017b] R-bundle-Bioconductor v3.6, MariaDB-connector-c v2.3.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
depends on #5552 (CMake)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2408.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/671b2cd707979d35ae3169f2e0d6105b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2004.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/19112d0a844d41319e15b26f2f97a9b2 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 1 out of 5 (2 easyconfigs in this PR)
rescomp2 - Linux centos 6.8, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.7.11
See https://gist.github.com/1903c3f1e5731f0e1f5f25da1e2287f5 for a full test report.

**Comment by verdurin** [comment]
```
../../../../src/gallium/auxiliary/.libs/libgallium.a(lp_bld_misc.o):(.data._ZTIN4llvm18MCJITMemoryManagerE[_ZTIN4llvm18MCJITMemoryManagerE]+0x10): undefined reference to `typeinfo for llvm::RuntimeDyld::MemoryManager'
../../../../src/gallium/auxiliary/.libs/libgallium.a(lp_bld_misc.o):(.data._ZTI26DelegatingJITMemoryManager[_ZTI26DelegatingJITMemoryManager]+0x10): undefined reference to `typeinfo for llvm::RTDyldMemoryManager'
../../../../src/gallium/drivers/swr/.libs/libmesaswr.a(libmesaswr_la-JitManager.o):(.data._ZTI8JitCache[_ZTI8JitCache]+0x10): undefined reference to `typeinfo for llvm::ObjectCache'
make[4]: *** [libGL.la] Error 1
make[4]: Leaving directory `/dev/shm/Mesa/17.2.4/intel-2017b/mesa-17.2.4/src/gallium/targets/libgl-xlib'
make[3]: *** [all-recursive] Error 1
make[3]: Leaving directory `/dev/shm/Mesa/17.2.4/intel-2017b/mesa-17.2.4/src/gallium'
make[2]: *** [all-recursive] Error 1
make[2]: Leaving directory `/dev/shm/Mesa/17.2.4/intel-2017b/mesa-17.2.4/src'
make[1]: *** [all] Error 2
make[1]: Leaving directory `/dev/shm/Mesa/17.2.4/intel-2017b/mesa-17.2.4/src'
make: *** [all-recursive] Error 1
```

**Comment by boegel** [comment]
@verdurin Can you check whether your LLVM installation was configured with `-DLLVM_ENABLE_RTTI=1 ` (see https://github.com/easybuilders/easybuild-easyconfigs/pull/5321/commits/1b43ffcf6cbe4c15424b4a3d02ab412a2a78e1f0, which was added later in #5321)?

**Comment by verdurin** [comment]
@boegel It wasn't - rebuilding now. I think you and/or @wpoely86 had already mentioned this.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 2 out of 4 (2 easyconfigs in this PR)
rescomp2 - Linux centos 6.8, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.7.11
See https://gist.github.com/ecf90c08c63aef4e2d365c2c4276875c for a full test report.

**Comment by boegel** [comment]
@verdurin You need to use EasyBuild v3.5.0 with the updated R easyconfigs, to avoid building a sub-optimal R installation (that doesn't correctly use BLAS/LAPACK provided by the toolchain).

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
rescomp2 - Linux centos 6.8, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.7.11
See https://gist.github.com/9b0a17d9ccd97f72fa098f685c85e893 for a full test report.

File: easybuild/easyconfigs/m/MariaDB-connector-c/MariaDB-connector-c-2.3.4-intel-2017b.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+description = "MariaDB Connector/C is used to connect applications developed in C/C++ to MariaDB and MySQL databases."
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+
+source_urls = ['https://downloads.mariadb.org/f/connector-c-%(version)s']
+sources = ['mariadb-connector-c-%(version)s-src.tar.gz']
+checksums = ['8beb0513da8a24ed2cb47836564c8b57045c3b36f933362f74b3676567c13abc']
+
+builddependencies = [('CMake', '3.10.1')]
+dependencies = []
```
Comment (vanzod) [inline easybuild/easyconfigs/m/MariaDB-connector-c/MariaDB-connector-c-2.3.4-intel-2017b.eb]: Is this necessary?

Comment (boegel) [reply easybuild/easyconfigs/m/MariaDB-connector-c/MariaDB-connector-c-2.3.4-intel-2017b.eb]: Hah, of course not, I was expected that a full blown `MariaDB` was going to be required here, but that's not the case. Will clean up, thanks

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**